libs/icons/scripts/cheatsheet-template.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
```

Comment:
testing info on how to view this page would be great

Reply:
It renders when you run the project with `yarn start`